auto-completion/bash/buku-completion.bash

```diff
@@ -16,6 +16,7 @@ _buku () {
         -d --delete
         --deep
         -e --export
+        --editor
```

Comment:
Hmm I am not sure If i should blindly add them in, since I can't really test them. 
will update it with -w and --write for bash completion